web/src/title/NewProjectMode.svelte

```diff
@@ -70,13 +75,14 @@
       window.alert(`Couldn't create project: ${err}`);
       return;
     }
-    loading = `Loading pre-clipped OSM area ${example}`;
     await loadProject(projectID);
     $mode = { mode: "add-neighbourhood" };
-    loading = "";
   }
 </script>
 
+<Loading {loading} progress={100} />
```

Comment:
Slightly weird, but two cases are possible from here -- directly start pulling from overpass and then create a project, or create a project file and load from it the normal way